arangosh/Restore/RestoreFeature.cpp

```diff
@@ -993,6 +1104,10 @@ void RestoreFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opt
 
   options->addOption("--input-directory", "input directory",
                      new StringParameter(&_options.inputPath));
+  
+  options->addOption("--cleanup-duplicate-attributes", "clean up duplicate attributes in input documents instead of making the restore operation fail (since v3.3.22 and v3.4.2)",
```

Comment:
@sleto-it We [recently added](https://github.com/arangodb/arangodb/pull/7701/files) `From v3.4.0` to another option. Should this here be changed to the same style (separate sentence), or should we change the other one to also be written in parentheses `(from v3.4.0)`?

```suggestion
  options->addOption("--cleanup-duplicate-attributes", "clean up duplicate attributes in input documents instead of making the restore operation fail. From v3.3.22 and v3.4.2",
```

Reply:
Can we stop putting version numbers into help strings?
It looks like this is not the right format for this at all.

Reply:
I agree (also see [here](https://github.com/arangodb/arangodb/pull/7701#issuecomment-445846471)). Any suggestions how to do this in a better way?

Reply:
Two options IMHO:
* not specify the version number at all (which improves readability, and is also consistent with all the "old" options for which there is no version number specified)
* add a seperate "introduced" version number for options, which is not part of the description, but can be used seperate, e.g. for generating the docs. Ideally we would restrict ourselves to a single version number here. IMHO it does not provide any benefit to show "added in v3.3.22 and v3.4.2". If you are in 3.3, v3.3.22 is the relevant information. If you are on 3.4, v3.4.2 is the only thing that is of interest.

Reply:
- Santo explained that the version numbers are needed in support, for end users as well as our staff. We may have to retroactively add version numbers for "old" options, but that's to be discussed.
- A separate field sounds good to me. With our current documentation, it would be sufficient to have a single version number. However, there are plans to change the documentation, so that everyone can use the latest docs even for older versions. Then a single version number would no longer suffice (and it's inconvenient today already if you have to compare e.g. 3.3 and 3.4 docs to find out what changed between versions - not every change is in the changelog).